e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -122,15 +99,60 @@ private Document getFirstDocumentIfItHasObjectNameOrElseNull(Document gpcAccessD
         return null;
     }
 
+    private void assertThatAcknowledgementToRequesterWasSent(Document ackToRequester) {
+        assertAll(
+            () -> assertThat(ackToRequester.get("messageId")).isNotNull(),
```

Comment:
(optional) there is also a SoftAssertionsExtension which can handle some of this boilderplate for you
https://assertj.github.io/doc/#assertj-core-junit5-soft-assertions